### ISSUE TITLE
Incompatible tensorflow version

### ISSUE THREAD

**Issue by junxiant** [nie-lang/DeepRectangling#4]
Hello,

Could you list the full requirements for this code? I'm getting tensorflow version errors even when using 1.13.1

**Comment by nie-lang** [comment]
My full experimental environment:

OS: Ubuntu
GPU: 2080Ti
CUDA:10.1
Driver version: 430.34
Anaconda env:
[environment.txt](https://github.com/nie-lang/DeepRectangling/files/8803961/environment.txt)

Hope it will help you.

**Comment by junxiant** [comment]
Thanks it works with that txt env

**Comment by AAaaronnn** [comment]
Hello,when i run the codes i met a question. It's "The passed save_path is not a valid checkpoint: ./checkpoints/pretrained_model/model.ckpt-100000".I've got no idea what it mean.Could you tell me how to solve this question?And I wanna train my dataset.I now have two perspectives of the pictures and want to stitch it together using codes, but don't know what does the "mask" mean,and what i can put in the "input".Thanks for your answers.

**Comment by junxiant** [comment]
You need to check if its asking for a "xxx.ckpt" or a path to the folder directory.

I only used this for inference, I did not train my own dataset, cant help you with this one.

**Comment by Sunhan-Ash** [comment]
@AAaaronnn Hello, I encountered the same problem as you. I don't know what "mask" is. Do you now know how "mask" is generated? can you help me?
